code/game/objects/items/weapons/tools/mods/mod_types.dm

```diff
@@ -693,7 +694,9 @@
 	tool_comp.tool_upgrades[UPGRADE_BULK] = rand(-1,2)
 	QDEL_NULL(spawn_type)
 	QDEL_NULL(CATCH)
+	price_tag += rand(0, 500)
```

Comment:
Art should be worth at least a little - even if it absolutely sucks, it can be exported as scrap.

It should be worth a good bit if you're lucky - after all, the Artist should be selling a lot of his spare art to the merchant.
```suggestion
	price_tag += rand(100, 1000)
```